CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.3.5
```

Comment:
It's not breaking changes but it's a pretty big feature. maybe 0.4?

Reply:
Whatever you want boss :wink:

Reply:
Also, can you please add these changes https://github.com/exthereum/ethereumex/pull/42  to CHANGELOG?

Reply:
ping ^